zap/src/main/java/org/zaproxy/zap/control/ExtensionFactory.java

```diff
@@ -325,6 +326,13 @@ private static void loadMessages(Extension ext) {
         }
     }
 
+    private static boolean isSame(ResourceBundle rb1, ResourceBundle rb2) {
+        if (rb1 == null || rb2 == null) {
+            return false;
+        }
+        return Objects.equals(rb1.getBaseBundleName(), rb2.getBaseBundleName());
```

Comment:
Why not string.equals if we are getting names?

Reply:
I'm not saying it's wrong I'm just curious.

Reply:
Technically they might return null.